package.json

```diff
@@ -53,9 +53,6 @@
   "lint-staged": {
     "*.{js,ts,css,md,yml}|**/syntaxes/*.json|completions.json": [
       "prettier --write"
-    ],
-    "*.{js,ts}": [
-      "eslint --fix"
```

Comment:
Any rationale for this change?

Reply:
I could not commit anything without removing it 😓 
So tech debt I would say, I think it was painful before, we need to figure something better, I think Soph can plus one here.